meerk40t/core/elements/elements.py

```diff
@@ -1190,13 +1190,7 @@ def load_persistent_op_list(self, name):
         op_tree = dict()
         for section in list(settings.derivable(name)):
             op_type = settings.read_persistent(str, section, "type")
-            # That should not happen, but it happens nonetheless...
-            # So recover gracefully
-            try:
-                op = Node().create(type=op_type)
-            except (AttributeError, RuntimeError, ValueError) as err:
-                print(f"That should not happen, but ops contained: '{op_type}' [{err}]")
-                continue
+            op = Node().create(type=op_type)
```

Comment:
This needs to check with the bootstrapping to make sure the type is a real node and that it correctly read.